From 4441540d20d278953af107465d10ac9f44bade32 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:06 -0500
Subject: [redhat] kabi: Adding symbol cpu_active_mask (kernel/cpu.c)

Message-id: <aaebe120777ff3d4c1161534f1b332cac332ac31.1418903403.git.poros@redhat.com>
Patchwork-id: 102242
O-Subject: [PATCH RHEL7.1 14/48] kabi: Adding symbol cpu_active_mask (kernel/cpu.c)
Bugzilla: 1160754
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160754
    https://bugzilla.redhat.com/show_bug.cgi?id=1160754

    Description:
    ============
    These are used to play nice with cpu hotplug functionality.
    Used in superpages.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/cpu_active_mask
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/cpu_active_mask
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/cpu_active_mask
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/cpu_active_mask

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/cpu_active_mask b/redhat/kabi/kabi-module/kabi_ppc64/cpu_active_mask
new file mode 100644
index 0000000..1f38b15
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/cpu_active_mask
@@ -0,0 +1,2 @@
+#1-
+0x7ff5383e cpu_active_mask vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/cpu_active_mask b/redhat/kabi/kabi-module/kabi_ppc64le/cpu_active_mask
new file mode 100644
index 0000000..1f38b15
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/cpu_active_mask
@@ -0,0 +1,2 @@
+#1-
+0x7ff5383e cpu_active_mask vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/cpu_active_mask b/redhat/kabi/kabi-module/kabi_s390x/cpu_active_mask
new file mode 100644
index 0000000..a2c47f7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/cpu_active_mask
@@ -0,0 +1,2 @@
+#1-
+0x04927208 cpu_active_mask vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/cpu_active_mask b/redhat/kabi/kabi-module/kabi_x86_64/cpu_active_mask
new file mode 100644
index 0000000..e0eb0db
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/cpu_active_mask
@@ -0,0 +1,2 @@
+#1-
+0x2a86f131 cpu_active_mask vmlinux EXPORT_SYMBOL
-- 
1.7.1